docs/tutorials/02_speed.ipynb

```diff
@@ -0,0 +1,1074 @@
+       "      <th></th>\n",
+       "      <th>Values</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>Simulation input shape (X)</th>\n",
+       "      <td>(500, 10)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Simulation output shape (y)</th>\n",
+       "      <td>(500, 5)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># hold-out set samples (test_set_size)</th>\n",
+       "      <td>100</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Do hyperparameter search (param_search)</th>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Type of hyperparameter search (search_type)</th>\n",
+       "      <td>random</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># sampled parameter settings (param_search_iters)</th>\n",
+       "      <td>20</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scale data before fitting (scale)</th>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scaler (scaler)</th>\n",
+       "      <td>StandardScaler</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction before fitting (reduce_dim)</th>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction method (dim_reducer)</th>\n",
+       "      <td>PCA</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Cross-validation strategy (cross_validator)</th>\n",
+       "      <td>KFold</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># parallel jobs (n_jobs)</th>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "5425122a01414644a20964646560515f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Initializing:   0%|          | 0/11 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Time taken: 21.6130108833313 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "start = time.time()\n",
+    "\n",
+    "em = AutoEmulate()\n",
+    "em.setup(X, y, n_jobs=5)\n",
+    "em.compare()\n",
+    "\n",
+    "end = time.time()\n",
+    "print(f\"Time taken: {end - start} seconds\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### 2) restrict the number of models using `model_subset` \n"
```

Comment:
```suggestion
    "### 2) restrict the range of models using `model_subset` \n
    
    Another approach is to limit the range of models by selecting a subset of relevant types based on your domain and problem expertise. This selection process typically considers factors such as the nature of the problem, data characteristics or the need for interpretability. By narrowing down the types of models, you can reduce the computational burden and focus on the most promising architectures for your specific task."
```

Once we have more specific guidance on how to select domain-appropriate models, we should link this here